_posts/ericabrody/2016-02-22-ericabrody-drawing-clinic2.md

```diff
@@ -0,0 +1,29 @@
+ ---
```

Comment:
Actually that link showed up in our class posts I think because Payal and I discussed while screensharing the content of my actual posted trinket. So this is a copy of the post I made for Tuesday which is available here http://silshack.github.io/spring2016/ericabrody-drawing-clinic.html

So after class I deleted drawing-clinic2, since I didn't want 2 posts of the same thing in our feed

Hope that makes sense
Erica

From: Elliott Hauser <notifications@github.com<mailto:notifications@github.com>>
Reply-To: silshack/spring2016 <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, February 25, 2016 at 10:48 AM
To: silshack/spring2016 <spring2016@noreply.github.com<mailto:spring2016@noreply.github.com>>
Cc: Erica Brody <ebrody@live.unc.edu<mailto:ebrody@live.unc.edu>>
Subject: Re: [spring2016] Erica's drawing program before peer clinic (#640)


In _posts/ericabrody/2016-02-22-ericabrody-drawing-clinic2.md<https://github.com/silshack/spring2016/pull/640#discussion_r54115177>:

> @@ -0,0 +1,29 @@
> + ---


I believe this space here is making your drawing program not show up on this post. I assume you and @payalpn<https://github.com/payalpn> used screenshare directly from the trinket but still both of you should've checked the post after merging it. Please work together to fix this when you can

-
Reply to this email directly or view it on GitHub<https://github.com/silshack/spring2016/pull/640/files#r54115177>.